src/nightly.js

```diff
@@ -66,38 +82,19 @@ module.exports = function*(argv) {
         yield gitutil.resetFromOrigin();
     })
 
-    //get SHAS from platforms
+    // Get SHAS from repos
     var SHAJSON = yield retrieveSha(repos);
 
-    //save SHAJSON in cordova-cli repo
-    yield repoutil.forEachRepo([cli], function*() {
-        //need to get the path to cordova-cli using executil
-        var cordovaclidir = process.cwd();
-        fs.writeFileSync((path.join(cordovaclidir, 'shas.json')), JSON.stringify(SHAJSON, null, 4), 'utf8', function(err) {
-            if (err) return console.log (err);
-        });
-
-    });
-
-    //Update platform references at cordova-lib/src/cordova/platformsConfig.json
-    var cordovalibdir;
-    yield repoutil.forEachRepo([cordovaLib], function*() {
-        //need to get the path to cordova-lib using executil
-        cordovalibdir = process.cwd();
-    });
-
-    yield updatePlatformsFile(path.join(cordovalibdir, 'src/cordova/platformsConfig.json'), SHAJSON);
-
-
     var currentDate = new Date();
     var nightlyVersion = '-nightly.' + currentDate.getFullYear() + '.' +
-                        currentDate.getMonth() + '.' + currentDate.getDate();
+                        pad(currentDate.getMonth() + 1) + '.' + pad(currentDate.getDate());
     var cordovaLibVersion;
     //update package.json version for cli + lib, update lib reference for cli
     yield repoutil.forEachRepo([cordovaLib, cli], function*(repo) {
         var dir = process.cwd();
         var packageJSON = require(dir+'/package.json');
-        packageJSON.version = versionutil.removeDev(packageJSON.version) + nightlyVersion;
+        packageJSON.version = versionutil.removeDev(packageJSON.version) + nightlyVersion +
+            '+' + SHAJSON[repo.id];
```

Comment:
> Generally, eight to ten characters are more than enough to be unique within a project

(from [Git book](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection#Short-SHA-1)) so stripping to 8 characters should be enough for our purposes